practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/validation/PractitionerValidationTests.java

```diff
@@ -0,0 +1,205 @@
+    }
+    
+    private static List<PractitionerAttributeConfiguration> PREFIX_EMPTY_REGEX_MAP;
+    static {
+        PREFIX_EMPTY_REGEX_MAP = new LinkedList<PractitionerAttributeConfiguration>();
+        PREFIX_EMPTY_REGEX_MAP.add(new PractitionerAttributeConfiguration("givenName", "Given Name", "^[a-zA-Z]{1,35}$"));
+        PREFIX_EMPTY_REGEX_MAP.add(new PractitionerAttributeConfiguration("familyName", "Family Name", "^[a-zA-Z]{1,35}$"));
+        PREFIX_EMPTY_REGEX_MAP.add(new PractitionerAttributeConfiguration("prefix", "Prefix", ""));
+    }
+    
+    private static List<PractitionerAttributeConfiguration> ALL_EMPTY_REGEX_MAP;
+    static {
+        ALL_EMPTY_REGEX_MAP = new LinkedList<PractitionerAttributeConfiguration>();
+        ALL_EMPTY_REGEX_MAP
+                .add(new PractitionerAttributeConfiguration("givenName", "Given Name", ""));
+        ALL_EMPTY_REGEX_MAP
+                .add(new PractitionerAttributeConfiguration("familyName", "Family Name", ""));
+        ALL_EMPTY_REGEX_MAP.add(new PractitionerAttributeConfiguration("prefix", "Prefix", ""));
+    }
+
+    private static List<PractitionerAttributeConfiguration> INCOMNPLETE_MAP;
+    static {
+        INCOMNPLETE_MAP = new LinkedList<PractitionerAttributeConfiguration>();
+        INCOMNPLETE_MAP
+                .add(new PractitionerAttributeConfiguration("givenName", "Given Name", ""));
+    }
+
+    private static List<PractitionerAttributeConfiguration> ERROR_MAP;
+    static {
+        ERROR_MAP = new LinkedList<PractitionerAttributeConfiguration>();
+        ERROR_MAP
+                .add(new PractitionerAttributeConfiguration("givenName", "Given Name",""));
+        ERROR_MAP
+                .add(new PractitionerAttributeConfiguration("familyName", "Family Name",""));
+        ERROR_MAP
+                .add(new PractitionerAttributeConfiguration("wrongname", "Given Name",""));
+    }
+    
+    @ParameterizedTest
+    @CsvSource({ ",,,Prefix", ",test,test,Prefix", "test,,test,Given Name", ",test,,Prefix", "null,null,null,Prefix", "test,null,test,Given Name" })
+    void TestValidate_WhenFieldsAreEmptyOrNull_ThrowsValidationException(
+            @ConvertWith(NullableConverter.class) String prefix, @ConvertWith(NullableConverter.class) String givenName,
+            @ConvertWith(NullableConverter.class) String familyName, @ConvertWith(NullableConverter.class) String expectedField) {
+        // Arrange
+        when(configurationProvider.getConfiguration()).thenReturn(new PractitionerConfiguration("person",
+            "Practitioner", ALL_REQUIRED_UNDER_35_MAP));
+        Practitioner practitioner = new Practitioner(prefix, givenName, familyName);
+        var practitionerValidation = new PractitionerValidation(configurationProvider); 
+
+        // Act + Assert
+        var result = practitionerValidation.validate(practitioner);
+        Assertions.assertFalse(result.isValid());
```

Comment:
Just FYI, purely preference it's recommended to use assertThat for assertions instead of almost all the assertion methods. https://dzone.com/articles/the-benefit-of-using-assertthat-over-other-assert